client/me/account-close/closed.jsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import EmptyContent from 'components/empty-content';
+import Spinner from 'components/spinner';
+import getPreviousRoute from 'state/selectors/get-previous-route';
+import isAccountClosed from 'state/selectors/is-account-closed';
+
+/**
+ * Style dependencies
+ */
+import './closed.scss';
+
+class AccountSettingsClosedComponent extends Component {
+	onClick = () => {
+		window.location = '/';
+	};
+
+	render() {
+		/* eslint-disable-next-line no-shadow */
+		const { isAccountClosed, previousRoute, translate } = this.props;
```

Comment:
Let's rename the `isAccountClosed` prop instead of shadowing the imported variable and having to disable the ESLint rule.

Reply:
Thanks, fixed! :)